src/models/document/document-content-tests/dc-general.test.ts

```diff
@@ -89,18 +89,18 @@ describe("DocumentContentModel", () => {
       title: "Image 1",
       insertRowInfo: {
         rowInsertIndex: 1,
-        rowDropIndex: 1,
-        rowDropLocation: "bottom"
+        rowDropId: textTile2RowId,
+        rowDropLocation: "top"
```

Comment:
I think the arguments in the previous version of this test didn't make a lot of sense (if you're dropping on row index 1 + "bottom" you'd think the insert index would be 2. So I changed it to "top".